flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentInternal.java

```diff
@@ -53,6 +54,14 @@ public interface TableEnvironmentInternal extends TableEnvironment {
      */
     Parser getParser();
 
+    /**
+     * Return a {@link OperationExternalExecutor} that provides method for executing operation in
+     * external implementation.
+     *
+     * @return initialized {@link OperationExternalExecutor}.
+     */
+    OperationExternalExecutor getOperationExternalExecutor();
```

Comment:
It seems this method is only called by the implementation itself. So no need to expose it?